src/components/reminder-form/reminder-form.tsx

```diff
@@ -1,18 +1,46 @@
-import React, { FormEvent } from "react";
+import React, { FormEvent, useState } from "react";
 
-const Form = ({}: {}) => {
+
+const Form = ({ }: {}) => {
+  const [taskName, setTaskName] = useState('');
+  const [recurring, setRecurring] = useState(false);
   const handleSubmit = (e: FormEvent<HTMLFormElement>): void => {
     e.preventDefault();
   };
   return (
     <form onSubmit={handleSubmit} className="form-group">
       <div>
-        <input type="text" className="form-control" placeholder="Add a remider" />
+        <input type="text" className="form-control" placeholder="Remider Title" />
+      </div>
+      <div className="card mt-3 text-secondary ">
+        <h6>What would you like the reminder to say?</h6>
+        <textarea id="custom-message" ></textarea>
+      </div>
+      <div className="mt-3">
+        <input type="text" className="form-control" placeholder="How would you like to be reminded?" />
+      </div>
+      <div className=" card mt-3 text-secondary">
+        <h6>When would you like to be reminded?</h6>
+        <input type="date"></input>
```

Comment:
You can do `<input ... />` instead of `<input ...></input>` if the component doesn't have children